add_me_to_your_PATH/julia_pod

```diff
@@ -148,7 +148,7 @@ JULIA_COMMAND="[${JULIA_COMMAND}]"
 echo $JULIA_COMMAND
 
 # generate a k8s secret name
-SECRET_NAME=$(echo "${AWS_USER_ID}" | tr '[:upper:]' '[:lower:]' | sed -e 's/[^a-z0-9]\+/-/g')
+SECRET_NAME=$(echo "${KUBERNETES_NAMESPACE}_GITHUB_TOKEN_SECRET" | tr '[:upper:]' '[:lower:]' | sed -e 's/[^a-z0-9]\+/-/g')
```

Comment:
Unfortunately, this will problematic as this was used to support multiple users in the same namespace. This should be a viable alternative:
```suggestion
SECRET_NAME=$(echo "${USER}_gh_pat" | tr '[:upper:]' '[:lower:]' | sed -e 's/[^a-z0-9]\+/-/g')
```